src/main/java/org/apache/maven/plugins/changes/ChangesReport.java

```diff
@@ -172,11 +173,9 @@ public class ChangesReport extends AbstractChangesReport {
     @Parameter(property = "changes.xmlPath", defaultValue = "src/changes/changes.xml")
     private File xmlPath;
 
-    private ReleaseUtils releaseUtils = new ReleaseUtils(getLog());
-
-    private CaseInsensitiveMap caseInsensitiveIssueLinkTemplatePerSystem;
+    private final ReleaseUtils releaseUtils = new ReleaseUtils(getLog());
```

Comment:
This could be a little wonky, depending on what the getLog does. If it depends on this object being initialized, it might not work like you expect. In general avoid calling instance methods in constructors and field initializers.

Reply:
fixed in other place also